src/permissions/rules/rules.service.spec.ts

```diff
@@ -141,4 +141,13 @@ describe('RulesService', () => {
     await service.loadRules(DATABASE_NAME);
     expect(() => service.getRulesForUser(normalUser)).toThrow(ReferenceError);
   });
+
+  it("should only return 'public' rules if no user object is passed", () => {
+    testPermission.data.default = [{ subject: 'Config', action: 'read' }];
+    const publicRule: DocumentRule = { subject: 'User', action: 'create' };
+    testPermission.data.public = [publicRule];
+
+    const result = service.getRulesForUser(undefined);
+    expect(result).toEqual([publicRule]);
```

Comment:
`default` role does not apply to unauthenticated users, right? Maybe we could add an `expect(result)not.toContain(testPermission.data.default);` to make that even more explicit